### DISCUSSION TITLE
Does MODFLOW automatically convert confined aquifers to submersible aquifers?

### DISCUSSION THREAD

**Discussion by Alvin-prc** [modflowpy/flopy#1944] [Q&A] [answered]
If there is a confined aquifer with the water level slightly above the top of the aquifer, and wells are pumping water causing the water level to drop below the top of the aquifer, the confined aquifer transforms into an unconfined aquifer.

When using Flopy, I noticed that both ss and sy need to be set in flopy.modflow.ModflowLpf. Does this imply that if I initially set laytyp=0, Flopy will automatically adjust laytyp to 1 when the water level falls below the top of the aquifer? Or, is it necessary to manually adjust the aquifer type?

Is there an example available that demonstrates this process?

Thank you for addressing my question.

**Comment by mbakker7** [comment, marked as answer]
If you set laytyp to 1 then MODFLOW2005 will adjust the saturated thickness
and the storage coefficient when the head drops below the top of the
aquifer. It will check and convert every time step automatically.

In MODFLOW6 you have to set it twice. Once  in the NPF package and once in
the STO package.

Hope this helps.

**Reply by Alvin-prc** [reply]
So "convertible" means an aquifer type that can switch between confined and unconfined based on the head, right?

**Comment by mbakker7** [comment]
That is correct!
